CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java

```diff
@@ -0,0 +1,190 @@
+			showHelp("The template file" + dFile.getAbsolutePath() + " does not exist!", options);
+			return;
+		}
+		
+		
+		File outFile = new File(outputFile);
+		if(!outFile.exists()){
+			showHelp("The template file" + outFile.getAbsolutePath() + " does not exist!", options);
+			return;
+		}
+		
+		String templateFileContents = com.navercorp.cubridqa.common.CommonUtils.getFileContent(templateFile);
+		
+		Properties dataProp = com.navercorp.cubridqa.common.CommonUtils.getProperties(dataFile);
+		Set<Object> set = dataProp.keySet();
+		Iterator<Object> it = set.iterator();
+		String key;
+		while (it.hasNext()) {
+			key = (String) it.next();
+			String value = dataProp.getProperty(key);
+			if(!com.navercorp.cubridqa.common.CommonUtils.isEmpty(value) && value.toLowerCase().startsWith("json_file:")){
+				String[] vals = value.split(":");
+				if(vals == null || vals.length < 2){
+					throw new Exception("Please confirm your key format " + value + "  in data file, key->json(key=json_file:data.txt)");
+				}
+				
+				String jsonFileName = vals[2];
+				File jsonData = new File(jsonFileName);
+				if(!jsonData.exists()){
+					throw new Exception("Please confirm your json data file " + jsonData.getAbsolutePath() +  " exists!");
+				}
+				
+				String jsonDataContent = com.navercorp.cubridqa.common.CommonUtils.getFileContent(jsonData.getAbsolutePath());
+				com.navercorp.cubridqa.common.CommonUtils.replace(templateFileContents, key, escape(jsonDataContent));
+				
+			}else if(!com.navercorp.cubridqa.common.CommonUtils.isEmpty(value) && value.toLowerCase().startsWith("file:")){
+				String[] vals = value.split(":");
+				if(vals == null || vals.length < 2){
+					throw new Exception("Please confirm your key format " + value + "  in data file, key->file(key=file:data.txt)");
+				}
+				
+				String dataFileName = vals[2];
+				File fdata = new File(dataFileName);
+				if(!fdata.exists()){
+					throw new Exception("Please confirm your data file " + fdata.getAbsolutePath() +  " exists!");
+				}
+				
+				String fDataContent = com.navercorp.cubridqa.common.CommonUtils.getFileContent(fdata.getAbsolutePath());
+				com.navercorp.cubridqa.common.CommonUtils.replace(templateFileContents, key, fDataContent);
+			}else{
+				com.navercorp.cubridqa.common.CommonUtils.replace(templateFileContents, key, value);
+			}
+		}
+		
+	}
+
+	public static String readJsonTemplate(String templateFilePath) throws IOException {
+		String templateContents = com.navercorp.cubridqa.common.CommonUtils.getFileContent(templateFilePath);
+		return templateContents;
+	}
+	
+	private static void showHelp(String error, Options options) {
+		if (error != null) {
+			System.out.println("Error: " + error);
+		}
+		HelpFormatter formatter = new HelpFormatter();
+		formatter.printHelp("MergeTemplate.sh <-t|--template-file> template-file <-d|--data> data-file <-o|--out> out-file", options);
+		System.out.println();
+	}
+
+	public static String escape(String string) {
```

Comment:
Move below to CommonUtils.java
public static String escape(String string)

And change 'escape' to 'genJson'